schema/table.go

```diff
@@ -200,9 +199,10 @@ func (t Table) Resolve(ctx context.Context, meta ClientMeta, syncTime time.Time,
 				}
 			}
 
+			totalResources++
```

Comment:
Incrementing when we send the result to the channel seems more clear to me than counting all the objects before the loop

Reply:
Maybe worth adding a separate log message for relations if we don't have one